src/pages/index.tsx

```diff
@@ -74,6 +75,11 @@ const IndexPage = () => {
                   API
                 </SmartLink>
               </Nav.Item>
+              <Nav.Item>
+                <Nav.Link className="text-primary" onClick={() => {
+                  startSandbox();
```

Comment:
@Neo-Zhixing we should replicate what we do on static sites to be able to turn off links to the sandbox while we upgrade